server/src/main/java/org/cloudfoundry/identity/uaa/oauth/openid/IdTokenCreator.java

```diff
@@ -134,7 +134,7 @@ private Map<String, List<String>> buildUserAttributes(UserAuthenticationData use
         }
 
         if (requestedAttributes && attributes == null) {
-            logger.debug(String.format("Requested id_token containing %s, but no saved attributes available for user with id:%s. Ensure storeCustomAttributes is enabled for origin:%s in zone:%s.", ClaimConstants.USER_ATTRIBUTES, user.getId(), user.getOrigin(), identityZoneManager.getCurrentIdentityZoneId()));
+            logger.debug("Requested id_token containing %s, but no saved attributes available for user with id:%s. Ensure storeCustomAttributes is enabled for origin:%s in zone:%s.".formatted(ClaimConstants.USER_ATTRIBUTES, user.getId(), user.getOrigin(), identityZoneManager.getCurrentIdentityZoneId()));
```

Comment:
log messages can be formatted with `{}` instead of `.formatted(`

Reply:
I know but this change was done by openrewrite tool , done before spring boot 3 update